crates/re_viewer/src/ui/view_textbox/scene.rs

```diff
@@ -0,0 +1,54 @@
+use re_arrow_store::LatestAtQuery;
+use re_log::warn_once;
+use re_log_types::component_types::{self};
+use re_query::{query_entity_with_primary, QueryError};
+
+use crate::{ui::SceneQuery, ViewerContext};
+
+// ---
+
+#[derive(Debug, Clone)]
+pub struct TextboxEntry {
+    pub body: String,
```

Comment:
```suggestion
    pub body: String, // TODO(#1887): avoid allocations
```

Reply:
Ahh, nice. That ticket also suggests:
```
We should also take the time to split TextEntry into two components: text and level.
```

Maybe we should just do that all now.

Reply:
It feels like the new view this PR introduces should just be a different view of the same component. The `TextLog` view shows text over time, while the new `TextBox` view shows the latest text(s) at the given time. That would make the difference a difference in blueprint, not in data or component type.

Reply:
(If only we had a way to choose the view in the blueprint 😅)

Having a marker-component to hint at the right View and different APIs for logging still seems important for user discoverability and ergonomics. These are very different use-cases. Forcing blueprint-interactions to differentiate them doesn’t feel like a great experience.